ranking_survey/index.html

```diff
@@ -0,0 +1,222 @@
+        align-items: center;
+      }
+
+      .center-modal {
+        position: relative;
+        background: #fff;
+        margin: 18px;
+        padding: 72px 24px 40px;
+        border-radius: 8px;
+        height: min-content;
+        width: 100%;
+        box-shadow: rgb(0 0 0 / 30%) 0px 0px 12.5px,
+          rgb(0 0 0 / 15%) 0px 0px 2.5px;
+      }
+
+      .center-modal .close-button {
+        position: absolute;
+        top: 0;
+        right: 0;
+        background: rgba(0, 0, 0, 0);
+        border: none;
+        z-index: 1;
+        display: flex;
+        justify-content: center;
+        flex-direction: column;
+        align-items: center;
+        /* Tip: Make your close-button relatively large so it's easy to click */
+        min-width: 48px;
+        min-height: 48px;
+      }
+
+      h1 {
+        font-family: Raleway;
+        font-size: 24px;
+        font-weight: 700;
+        line-height: 28px;
+        letter-spacing: 0px;
+        text-align: left;
+        color: #051b2c;
+      }
+
+      h2 {
+        font-family: Raleway;
+        font-size: 16px;
+        font-weight: 500;
+        line-height: 24px;
+        letter-spacing: 0px;
+        text-align: left;
+        color: #28333e;
+        margin-bottom: 22px;
+      }
+
+      @media screen and (min-width: 480px) {
+        .center-modal {
+          height: 80%;
+        }
+      }
+
+      .survey-form ul {
+        display: grid;
+        grid-auto-flow: column;
+        margin-bottom: 14px;
+      }
+
+      .survey-form li button {
+        background: #cae4fa;
+        border: 1px solid #aad4f7;
+        border-radius: 4px;
+        width: 45px;
+        height: 35px;
+      }
+
+      .legend {
+        display: flex;
+        justify-content: space-between;
+        color: #74808b;
+        font-family: "Raleway";
+        font-style: normal;
+        font-weight: 500;
+        font-size: 14px;
+        line-height: 24px;
+      }
+    </style>
+  </head>
+
+  <body>
+    <div class="center-modal">
+      <div class="close-button" data-onesignal-unique-label="close-button">
+        <svg
+          width="10"
+          height="10"
+          preserveAspectRatio="none"
+          viewBox="0 0 8 8"
+          fill="none"
+          xmlns="http://www.w3.org/2000/svg"
+        >
+          <path
+            d="M7.80309 1.14768C8.06564 0.885137 8.06564 0.459453 7.80309 0.196909C7.54055 -0.0656362 7.11486 -0.0656362 6.85232 0.196909L4 3.04923L1.14768 0.196909C0.885137 -0.0656362 0.459453 -0.0656362 0.196909 0.196909C-0.0656362 0.459453 -0.0656362 0.885137 0.196909 1.14768L3.04923 4L0.196909 6.85232C-0.0656362 7.11486 -0.0656362 7.54055 0.196909 7.80309C0.459453 8.06564 0.885137 8.06564 1.14768 7.80309L4 4.95077L6.85232 7.80309C7.11486 8.06564 7.54055 8.06564 7.80309 7.80309C8.06564 7.54055 8.06564 7.11486 7.80309 6.85232L4.95077 4L7.80309 1.14768Z"
+            fill="#111111"
+          />
+        </svg>
+      </div>
+      <h1>Hi, Olivia</h1>
```

Comment:
maybe a good place to recommend using tag substitution here? `<h1> Hi, {{ name | default: 'There' }}</h1>`?